src/services/developer/developer.svc.coffee

```diff
@@ -27,14 +27,7 @@ angular
 
```

Comment:
It's working even better than before: now if you try to add a template in the workspace without having defined a proper component related to it, it won't break ;)